testing/eigen/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=eigen
+pkgver=3.3.4
+pkgrel=0
+pkgdesc="Eigen is a C++ template library for linear algebra"
+url="http://eigen.tuxfamily.org/index.php?title=Main_Page"
+arch="all"
+license="MPL2"
+depends=""
+makedepends=""
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::http://bitbucket.org/eigen/$pkgname/get/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-eigen-5a0156e40feb"
+
+build() {
+        return 0
+}
+
+package() {
+        mkdir -p "$pkgdir"/usr/include
+        cp -r "$builddir"/Eigen "$pkgdir"/usr/include
+        cp -r "$builddir"/unsupported "$pkgdir"/usr/include
```

Comment:
I understand that, but why it’s named `unsupported`? This is quite general name, it should be inside `/usr/include/Eigen` directory… if it _really_ belongs to `/usr/include`, there are even CMakeList.txt files, tests etc.